kroxylicious-filter-test-support/src/main/java/io/kroxylicious/test/assertj/HeaderAssert.java

```diff
@@ -28,13 +34,13 @@ public static HeaderAssert assertThat(Header actual) {
     private AbstractStringAssert<?> key() {
         var existingDescription = descriptionText();
         return Assertions.assertThat(actual.key())
-                .describedAs(existingDescription + " key");
+                .describedAs(existingDescription + KEY_SUFFIX);
```

Comment:
IIRC `describedAs` supports `printf`-style format strings, so we should probably use that rather than doing string concat.